docs/notebooks/dispersion/hollweg_dispersion.ipynb

```diff
@@ -0,0 +1,968 @@
+    "\n",
+    "## Contents:\n",
+    "\n",
+    "1. [Wave propogating at nearly 90 degrees](#Wave-propogating-at-nearly-90-degrees)\n",
+    "2. [Hollweg 1999 and Bellan 2012 comparsion](#Hollweg-1999-and-Bellan-2012-comparison)\n",
+    "3. [Reproduce Figure 2 from Hollweg 1999](#Reproduce-Figure-2-from-Hollweg-1999)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "import astropy.units as u\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import warnings\n",
+    "\n",
+    "from astropy.constants.si import c\n",
+    "from matplotlib.ticker import MultipleLocator\n",
+    "\n",
+    "from plasmapy.dispersion.analytical.two_fluid_ import two_fluid\n",
+    "from plasmapy.dispersion.numerical.hollweg_ import hollweg\n",
+    "from plasmapy.formulary.frequencies import gyrofrequency, plasma_frequency, wc_, wp_\n",
+    "from plasmapy.formulary.lengths import inertial_length\n",
+    "from plasmapy.formulary.speeds import Alfven_speed, cs_, ion_sound_speed, va_\n",
+    "from plasmapy.particles import Particle\n",
+    "\n",
+    "warnings.filterwarnings(action=\"ignore\")\n",
+    "\n",
+    "plt.rcParams[\"figure.figsize\"] = [10.5, 0.56 * 10.5]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Wave propogating at nearly 90 degrees\n",
+    "\n",
+    "Below we define the required parameters to compute the wave frequencies."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# define input parameters\n",
+    "inputs = {\n",
+    "    \"k\": np.logspace(-7, -2, 300) * u.rad / u.m,\n",
+    "    \"theta\": 89 * u.deg,\n",
+    "    \"n_i\": 5 * u.cm ** -3,\n",
+    "    \"B\": 1.10232e-8 * u.T,\n",
+    "    \"T_e\": 1.6e6 * u.K,\n",
+    "    \"T_i\": 4.0e5 * u.K,\n",
+    "    \"ion\": Particle(\"p+\"),\n",
+    "}\n",
+    "\n",
+    "# a few useful plasma parameters\n",
+    "params = {\n",
+    "    \"n_e\": inputs[\"n_i\"] * abs(inputs[\"ion\"].charge_number),\n",
+    "    \"cs\": ion_sound_speed(\n",
+    "        inputs[\"T_e\"],\n",
+    "        inputs[\"T_i\"],\n",
+    "        inputs[\"ion\"],\n",
+    "    ),\n",
+    "    \"va\": Alfven_speed(\n",
+    "        inputs[\"B\"],\n",
+    "        inputs[\"n_i\"],\n",
+    "        ion=inputs[\"ion\"],\n",
+    "    ),\n",
+    "    \"wci\": gyrofrequency(inputs[\"B\"], inputs[\"ion\"]),\n",
+    "}\n",
+    "params[\"lpe\"] = inertial_length(params[\"n_e\"], \"e-\")\n",
+    "params[\"wpe\"] = plasma_frequency(params[\"n_e\"], \"e-\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The computed wave frequencies ($rad/s$) are returned in a dictionary with the keys representing\n",
```

Comment:
```suggestion
    "The computed wave frequencies (rad/s) are returned in a dictionary with the keys representing\n",
```

[According to the documentation guide, physical units should not be formatted as math.](https://docs.plasmapy.org/en/latest/development/doc_guide.html#documentation-guidelines)